src/components/SubscriptAvatar.tsx

```diff
@@ -49,23 +62,23 @@ function SubscriptAvatar({size, backgroundColor, mainAvatar, secondaryAvatar, no
         <View style={[containerStyle, noMargin ? styles.mr0 : {}]}>
             <UserDetailsTooltip
                 shouldRender={showTooltip}
-                accountID={lodashGet(mainAvatar, 'id', -1)}
+                accountID={mainAvatar?.id ?? -1}
                 icon={mainAvatar}
             >
                 <View>
                     <Avatar
                         containerStyles={StyleUtils.getWidthAndHeightStyle(StyleUtils.getAvatarSize(size || CONST.AVATAR_SIZE.DEFAULT))}
-                        source={mainAvatar.source}
+                        source={mainAvatar?.source}
                         size={size || CONST.AVATAR_SIZE.DEFAULT}
```

Comment:
if we are setting size with `CONST.AVATAR_SIZE.DEFAULT,` do we need to check and use the Const here?

Reply:
@gedu good point, I think we don't need it